test_conformance/math_brute_force/main.cpp

```diff
@@ -813,24 +810,6 @@ int main(int argc, const char *argv[])
     else if (gStopOnError)
         vlog("Stopping at first error.\n");
 
-    if (gMeasureTimes)
-    {
-        vlog("%s times are reported at right (cycles per element):\n",
-             method[gReportAverageTimes]);
-        vlog("\n");
-        if (gSkipCorrectnessTesting)
-            vlog("   \t               ");
-        else
-            vlog("   \t                                        ");
-        if (gWimpyMode) vlog("   ");
-        for (int i = gMinVectorSizeIndex; i < gMaxVectorSizeIndex; i++)
-            vlog("\t  float%s", sizeNames[i]);
-    }
-    else
-    {
```

Comment:
Although I don't especially agree with the use of "\t" in log output, as it generally doesn't serve the intended purpose of aligning output, was this intended?  Shouldn't the code in the "else" branch be made unconditional, instead?

Reply:
AFAICT, this is to accommodate the difference in string length between "passed" and "Wimp pass". There should have been a comment explaining this, but it's reasonable not to add one right now. However, this difference in string lengths still appears to apply.

Reply:
Right, good catch! I've re-introduced the else branch unconditionally. The output format can be further improved later if desired.

Reply:
Agreed. It seems plausible to me that there was a general accommodation made for the measurement info made in the formatting of the rest of the output, which is no longer needed - but it's best not to change this all in one go.